.php_cs.dist

```diff
@@ -43,10 +43,9 @@ $rules = [
     '@PHP71Migration:risky' => true,
     'compact_nullable_typehint' => true,
     'void_return' => null,
-    // To be tested before insertion:
-//    'strict_comparison' => true,
-//    'strict_param' => true,
-//    'php_unit_strict' => true,
+    'strict_comparison' => true,
```

Comment:
this is creating a bunch of problems in the tests. some might indicate sloppy code, but others might be plane wrong (assertSame vs assertEquals). can we kick this option again?

Reply:
No we should fix the tests, we did this across all other bundles

Reply:
agree for the numeric things, but where we have objects that have the same value but are not the identical object, why would we not use assertEquals over some hack to avoid doing assertSame?

Reply:
If it is not the identical object we cannot use assertSame obviously, but in this case asserEquals is wrong too, in this case we should assert, what we want to really assert, for example a return value of a getter

WDYT?

cc @greg0ire

Reply:
I agree with you, using assertEquals with object feels hackish to me, just like I wouldn't use `==` to compare two dates (or anything, really). For dates, we could check that the difference between them is zero seconds like adviced here: https://github.com/sebastianbergmann/phpunit/issues/2456#issuecomment-277692041

Reply:
Currently I don’t have that much time, but I fixed all the other bundles (with @greg0ire) and would fix the tests here, too when I find some time...